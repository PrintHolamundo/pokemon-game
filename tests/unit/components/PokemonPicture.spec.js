import { shallowMount } from '@vue/test-utils'
import PokemonPicture from '@/components/PokemonPicture'


describe('PokemonPicture', () => {


    test('debe de hacer match con el snapshop', () => {
        const wrapper = shallowMount(PokemonPicture, {
            props: {
                pokemonId: 1,
                showpokemon: false,
            }
        })
        expect(wrapper.html()).toMatchSnapshot()

    })

    test('debe de mostrar la imagen oculta y el pokemon 100', () => {

    })

    test('debe de mostrar el pokemon si showpokemon:true', () => {

    })
})