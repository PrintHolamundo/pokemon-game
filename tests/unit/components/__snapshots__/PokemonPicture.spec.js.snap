// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PokemonPicture debe de hacer match con el snapshop 1`] = `<div class="pokemon-container" showpokemon="false"><img src="https://raw.githubusercontent.com/PokeAPI/sprites/master/sprites/pokemon/other/dream-world/1.svg" class="hidden-pokemon" alt="pokemon"></div>`;
